docs/platform-external/use-case-oci-upi.md

```diff
@@ -0,0 +1,1829 @@
+
+oci compute instance launch \
+    --hostname-label "bootstrap" \
+    --display-name "bootstrap" \
+    --availability-domain "$AVAILABILITY_DOMAIN" \
+    --fault-domain "FAULT-DOMAIN-1" \
+    --compartment-id $COMPARTMENT_ID_OPENSHIFT \
+    --subnet-id $SUBNET_ID_PUBLIC \
+    --nsg-ids "[\"$NSG_ID_CPL\"]" \
+    --shape "$INSTANCE_SHAPE" \
+    --shape-config "{\"memoryInGBs\":16.0,\"ocpus\":8.0}" \
+    --source-details "{\"bootVolumeSizeInGBs\":120,\"bootVolumeVpusPerGB\":60,\"imageId\":\"${IMAGE_ID}\",\"sourceType\":\"image\"}" \
+    --agent-config '{"areAllPluginsDisabled": true}' \
+    --assign-public-ip True \
+    --user-data-file "./user-data-bootstrap.json" \
+    --defined-tags "{\"$CLUSTER_NAME\":{\"role\":\"master\"}}"
+```
+
+!!! tip "Helper - OCI CLI documentation"
+    - [`oci compute instance launch`](https://docs.oracle.com/en-us/iaas/tools/oci-cli/3.29.1/oci_cli_docs/cmdref/compute/instance/launch.html)
+    - [`oci compute shape list`](https://docs.oracle.com/en-us/iaas/tools/oci-cli/3.29.1/oci_cli_docs/cmdref/compute/shape/list.html)
+
+!!! tip "Follow the bootstrap process"
+    You can SSH to the node and follow the bootstrap process:
+    `journalctl -b -f -u release-image.service -u bootkube.service`
+
+
+- Discover the Load Balancer's backend sets and the Bootstrap Instance IDs:
+
+```sh
+BES_API_NAME=$(oci nlb backend-set list --network-load-balancer-id $NLB_ID | jq -r '.data.items[] | select(.name | endswith("api")).name')
+BES_MCS_NAME=$(oci nlb backend-set list --network-load-balancer-id $NLB_ID | jq -r '.data.items[] | select(.name | endswith("mcs")).name')
+
+INSTANCE_ID_BOOTSTRAP=$(oci compute instance list  -c $COMPARTMENT_ID_OPENSHIFT | jq -r '.data[] | select((.["display-name"]=="bootstrap") and (.["lifecycle-state"]=="RUNNING")).id')
+
+test -z $INSTANCE_ID_BOOTSTRAP && echo "ERR: Bootstrap Instance ID not found=[$INSTANCE_ID_BOOTSTRAP]. Try again."
+```
+
+!!! tip "Helper - OCI CLI documentation"
+    - [`oci nlb network-load-balancer list`]()
+    - [`oci nlb backend-set list`]()
+    - [`oci compute instance list`]()
+
+- Add the bootstrap to the Load Balancer's backend set API:
+
+```sh
+# oci nlb backend-set update --generate-param-json-input backends
+cat <<EOF > ./nlb-bset-backends-api.json
+[
+  {
+    "isBackup": false,
+    "isDrain": false,
+    "isOffline": false,
+    "name": "${INSTANCE_ID_BOOTSTRAP}:6443",
+    "port": 6443,
+    "targetId": "${INSTANCE_ID_BOOTSTRAP}"
+  }
+]
+EOF
+
+# Update API Backend Set
+oci nlb backend-set update --force \
+  --backend-set-name $BES_API_NAME \
+  --network-load-balancer-id $NLB_ID \
+  --backends file://nlb-bset-backends-api.json \
+  --wait-for-state SUCCEEDED
+```
+
+!!! tip "Helper - OCI CLI documentation"
+    - [`oci nlb backend-set update`](https://docs.oracle.com/en-us/iaas/tools/oci-cli/3.29.1/oci_cli_docs/cmdref/nlb/backend-set/update.html)
+
+- Add the bootstrap to the Load Balancer's backend set MCS[Machine Config Server]:
```

Comment:
i don't think we should use the square braces here, maybe just spell out "Machine Config Server" instead of adding "MCS"

alternatively, if we use MCS in other places we should make sure to put "Machine Config Server (MCS)" in the first usage for this document.